packages/utilities-react/src/StepFlow/StepContext.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useContext, createContext, ReactNode, useState } from 'react';
+import { StepContextType } from './types';
+
+const StepContext = createContext<StepContextType | undefined>(undefined);
+
+interface StepProviderProps {
+  children?: ReactNode;
+}
+
+export const StepProvider: React.FC<StepProviderProps> = ({ children }) => {
+  const [totalSteps, setTotalSteps] = useState<number>();
+  const [currentStep, setCurrentStep] = useState(1);
```

Comment:
`currentStep` is initialized to `1` but `totalSteps` is initialized to `undefined`. Seem like both should be initialized to numbers or neither should since a user can't be on step `1` of `undefined` steps.

Reply:
Agreed